test/integration/user-management-api/account_plan_features_test.rb

```diff
@@ -25,52 +26,37 @@ def setup
   end
 
   test 'not found account_plan replies 404' do
-    get(admin_api_account_plan_features_path(0),
-             :provider_key => @provider.api_key, :format => :xml)
+    get admin_api_account_plan_features_path(0), params: { provider_key: @provider.api_key, format: :xml }
     assert_xml_404
   end
 
   test 'enable new feature' do
-    feat = FactoryBot.create :feature, :featurable => @provider
-
-    post(admin_api_account_plan_features_path(@account_plan.id),
-              :feature_id => feat.id,
-              :provider_key => @provider.api_key, :format => :xml)
-
+    post admin_api_account_plan_features_path(@account_plan.id), params: { feature_id: feat.id, provider_key: @provider.api_key, format: :xml }
     assert_response :success
 
     xml = Nokogiri::XML::Document.parse(@response.body)
-
-    assert xml.xpath('.//feature/id').children.first.text == feat.id.to_s
+    assert_equal xml.xpath('.//feature/id').children.first.text, feat.id.to_s
   end
 
   test 'enabling feature not in account replies 404' do
-    feature_not_in_account = FactoryBot.create(:feature,
-                                     :featurable => FactoryBot.create(:provider_account),
-                                     :scope => "AccountPlan")
+    feature_not_in_account = FactoryBot.create(:feature, featurable: FactoryBot.create(:provider_account), scope: "AccountPlan")
 
-    post(admin_api_account_plan_features_path(@account_plan.id),
-              :feature_id => feature_not_in_account.id,
-              :provider_key => @provider.api_key, :format => :xml)
+    post admin_api_account_plan_features_path(@account_plan.id), params: { feature_id: feature_not_in_account.id, provider_key: @provider.api_key, format: :xml }
     assert_xml_404
   end
 
   pending_test 'enable existing feature'
 
   test 'disable feature' do
-    feat = FactoryBot.create :feature, :featurable => @provider
-
-    post(admin_api_account_plan_features_path(@account_plan.id),
-              :feature_id => feat.id,
-              :provider_key => @provider.api_key, :format => :xml)
-
+    post admin_api_account_plan_features_path(@account_plan.id), params: { feature_id: feat.id, provider_key: @provider.api_key, format: :xml }
```

Comment:
Just curious, how are we disabling anything here? In docs I see this as creating.
![image](https://user-images.githubusercontent.com/1408661/139677527-c87aa469-7c73-4148-ab5b-6633a9c0f949.png)

Reply:
Well create and enable are basically synonym in this context. I think doc is a bit confusing like you, but it is to create a link between a feature and a plan

Reply:
I was referring to test name `disable feature`. But I don't see us disable anything. That's why I asked. Maybe test name better be changed?

> doc is a bit confusing like you

Apologizes, fwiw my gf thinks the same :angel:

Reply:
It's very confusing yes, I even debugged a little and didn't found anything "disabled"... ultimately decided not to change it and just fix the deprecation. We can take a look at it again in the future.